aten/src/ATen/native/native_functions.yaml

```diff
@@ -4193,28 +4187,22 @@
 - func: digamma.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   named_guard: False
   dispatch:
-    CPU: _digamma_out_cpu
-    CUDA: legacy::cuda::_th_digamma_out
+    CPU: digamma_out
+    CUDA: digamma_out
 
 - func: digamma(Tensor self) -> Tensor
   named_guard: False
   variants: method, function
-  dispatch:
-    CPU: digamma
-    CUDA: legacy::cuda::_th_digamma
 
 - func: polygamma.out(int n, Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   named_guard: False
   dispatch:
     CPU: polygamma_out
-    CUDA: legacy::cuda::_th_polygamma_out
+    CUDA: polygamma_out
```

Comment:
I don't think we need these dispatch options anymore because they're dispatching to the same function. You should be able to delete the `dispatch:`

Reply:
Fixed